widget/embedded/src/components/Layout/Layout.styles.ts

```diff
@@ -36,10 +47,17 @@ export const Content = styled('div', {
   overflowY: 'auto',
 });
 export const Footer = styled('div', {
+  padding: '0 $20 $10',
+
   '& .footer__content': {
-    padding: '$0 $20',
+    marginBottom: '$12',
```

Comment:
Why we need this change? 🤔

Reply:
fixed

Reply:
Why it should be margin instead of padding? it's generally discouraged during it's potential affect on its container's layout.

Reply:
This code has been restored